### DISCUSSION TITLE
AI判斷力

### DISCUSSION THREAD

**Discussion by xipeiguo** [1101-GS4719-C/Python-Community-GS4719-C#2077] [General]
https://news.pts.org.tw/article/682913
有鑑於ai的發展，人們在遇到困難時常會開始詢問chatgpt尋求解答，然而在文章中也指出chatgpt亦有回答錯誤之時，我認為就算科技在如何發達，人們都不應盲目地相信chatgpt的回答，應保有自己的判斷能力，就算問題並不在自身的專業領域中，也應以chatgpt為輔，自己仍須努力依靠個人尋求解答。

**Comment by peresonacifer** [comment]
在運用ChatGPT尋找答案時，他很常會用條列式的方式講非常多的內容，這時要耐心的去看完他，盡可能的多去查證，或是針對自己覺得有問題的部分再去問ChatGPT，給他另一種可能去思考，他可能就會回說他說錯並再做更改，程式方面，就是直接跑看看行不行，如果不行，可能要考慮重問或請他修正。

**Comment by ds0w0** [comment]
chatgpt有其資料的侷限性，因此使用者必須擁有一定程度的媒體識讀能力，在執行GPT時應盡可能添加條件或給予資料，提供GPT做出更準確地回答。我認為AI只能幫助人們整理資料和生成文本，並不具備決策能力，因此查證和進行人為的修改與校稿是每個使用者的義務。

**Comment by Jerryguo888** [comment]
隨著AI技術的進步，許多人在遇到問題時會尋求ChatGPT的幫助。然而，ChatGPT有時也可能提供錯誤的回答，因此我們可以在問題上多加一些描述來確保ChatGPT能提供正確的答案。
